### PR TITLE
CHAOS-1928: adds chaos 0.14.1 release notes

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Link to Preview**](https://649ae9ec90855a0ecece52bd--harness-developer.netlify.app/release-notes/whats-new)
This PR:
- Adds release notes
- Removes FF notes on gameday and SRM topics, and adds notes "available to all" in release notes
- Moves SRM topic/graphics to "Integrations" folder in navigation
- Adds redirect for SRM topic to new location

### CODE REVIEW

File: release-notes/chaos-engineering.md [hunk 1]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
 ```mdx-code-block
 <Tabs>
   <TabItem value="What's new">
 ```
 
-* Added a new Linux chaos fault, Disk Fill, which fills up the available disk space at a given system path for a specific duration. (CHAOS-1419)
+* While upgrading a namespace-scoped chaos infrastructure, users will now be shown the command for upgrading CRDs as well. (CHAOS-1846)
 
-* To help users select the right infrastructure for their use case, the Chaos Infrastructures UI screen has been enhanced to show supported faults by different chaos infrastructure categories. (CHAOS-1811)
+* Upgraded the database to update the index in the linuxInfrastructures collection. (CHAOS-1836)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:27]: We can remove this one!

File: release-notes/chaos-engineering.md [hunk 2]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
 
-* The database was upgraded to update the index in linuxInfrastructures collection. (CHAOS-1836)
+* We now show all steps in the experiment details pipeline diagram. (CHAOS-1817)
+
+    Previously when users triggered chaos experiments, the execution graph generated step nodes progressively as the experiments executed. Now, the execution graph shows all step nodes after experiments start execution. The nodes yet to start remain in a pending state. 
+
+* Previously, when users connected a ChaosHub, CE cloned the whole Github repository. This caused storage issues if the repository was very large, or users were using the same repository for multiple purposes. This has been enhanced so that CE clones only a single branch provided by users. (CHAOS-1722)
+
+* Added a feature that removes chaos entities when there's a delete event for a project, org or account. Also subscribed to entity-crud redis stream to listen for deletion events. (CHAOS-1143)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:35]:
```suggestion
* Enhanced the chaos entities deletion functionality to delete all chaos entities associated to a project, org or account when there's a delete event for a project, org or account respectively. (CHAOS-1143)
```

File: release-notes/chaos-engineering.md [hunk 3]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
 ```
 
-* The Chaos Faults screen in ChaosHub was crashing when the **Platform** field was missing in the faults metadata file. This issue has been fixed. (CHAOS-1841)
+* (CHAOS-1917) _XXX ISSUE IN PROGRESS XXX_
+
+* **Probe mode** has now been made a required field. (CHAOS-1882)
+    
+    _XXX WHERE IS THIS FIELD? XXX_ 
+
+* The **Discard** button is now disabled if there are no changes in an experiment. (CHAOS-1878)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:54]: Enhanced the Discard button on Chaos Studio to be disabled if there are no changes in an experiment.

File: release-notes/chaos-engineering.md [hunk 4]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* The stop workflow feature wasn't able to stop experiments in the case of namespace-scoped chaos infrastructures. This issue has been resolved and the stop workflow now works as expected. (CHAOS-1778)
+
+* The **Chaos** Continuous Delivery (CD) step has been updated to handle new experiment and pipeline statuses. (CHAOS-733)
+
+* Added support for new experiment run statuses in the **Chaos** CD step. (CHAOS-1210)
+
+* Added logic to handle parallel experiments in the experiment manifest. (CHAOS-1208)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:62]: Fixed the issue when CD step was not showing parallel faults even when the selected experiment was having multiple parallel experiments

File: release-notes/chaos-engineering.md [hunk 5]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* **Probe mode** has now been made a required field. (CHAOS-1882)
+    
+    _XXX WHERE IS THIS FIELD? XXX_ 
+
+* The **Discard** button is now disabled if there are no changes in an experiment. (CHAOS-1878)
+
+* The stop workflow feature wasn't able to stop experiments in the case of namespace-scoped chaos infrastructures. This issue has been resolved and the stop workflow now works as expected. (CHAOS-1778)
+
+* The **Chaos** Continuous Delivery (CD) step has been updated to handle new experiment and pipeline statuses. (CHAOS-733)
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:58]: Fixed the issue when aborting an experiment running as part of a Pipeline, pipeline step used to show `All your faults executed without an issue`. Now it shows correct details based on experiment execution.

File: release-notes/chaos-engineering.md [hunk 6]
```diff
@@ -15,31 +15,52 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
-* This release does not include any early access features.
+This release does not include any early access features.
 
 ```mdx-code-block
   </TabItem>
   <TabItem value="Fixed issues">
 ```
 
-* The Chaos Faults screen in ChaosHub was crashing when the **Platform** field was missing in the faults metadata file. This issue has been fixed. (CHAOS-1841)
+* (CHAOS-1917) _XXX ISSUE IN PROGRESS XXX_
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:48]: The issue has been fixed when generating chaos infrastructures manifest with NodeSelectors/Tolerations was resulting into having first letter of key/value pairs in capital.

File: release-notes/chaos-engineering.md [hunk 7]
```diff
@@ -15,31 +15,55 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
...
+
+* When adding or updating a step in a chaos experiment, in the Probes tab, the **Probe mode** field is now required. (CHAOS-1882)
+    
+* The **Discard** button in Chaos Studio is now disabled if there are no changes in an experiment. (CHAOS-1878)
+
+* The stop workflow feature wasn't able to stop experiments in the case of namespace-scoped chaos infrastructures. This issue has been resolved and the stop workflow now works as expected. (CHAOS-1778)
+
+* There was an issue where if the user aborted an experiment running as part of a pipeline, the pipeline step displayed `All your faults executed without an issue`. This has been fixed, and the correct details are now displayed based on the experiment execution. (CHAOS-733)
+
+* Added support for new experiment run statuses in the **Chaos** Continuous Delivery (CD) step. (CHAOS-1210)
```
Comment (douglas-j-bothwell) [inline release-notes/chaos-engineering.md:63]: Why is this a Fixed Issue? Sounds more like a feature to me.

**Review by douglas-j-bothwell — CHANGES_REQUESTED** [review]
One comment and a few remaining to-do's:
1. The date in the frontmatter needs updating **date: 2023-06-05T10:00**
2. The What's New page needs to updated with the new CE  features

File: release-notes/chaos-engineering.md (hunk 7 above)
Comment (StrawberryPad) [reply release-notes/chaos-engineering.md:63]: Chaos is not part of the core release so it doesn't get space in "What's New". Fixed the rest.

File: release-notes/chaos-engineering.md [hunk 8]
```diff
@@ -15,31 +15,60 @@ Review the notes below for details about recent changes to Harness Chaos Enginee
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest - June 12, 2023, version 0.13.5
+## Latest - June 27, 2023, version 0.14.1
```
Comment (Jonsy13) [inline release-notes/chaos-engineering.md:18]:
```suggestion
## Latest - June 28, 2023, version 0.14.1
```

**Review by Jonsy13 — DISMISSED** [review]
Minor comment, looks good apart from that! Thanks @StrawberryPad 🚀

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**